MerlinAU.sh

```diff
@@ -5865,9 +6502,14 @@ _advanced_options_menu_()
                else _InvalidMenuSelection_
                fi
                ;;
-           bt) if [ "$fwInstalledBaseVers" -le 3004 ] && \
+           bt) if echo "$PRODUCT_ID" | grep -q "^TUF-"
+               then _ChangeBuildType_TUF_
+               elif [ "$fwInstalledBaseVers" -le 3004 ]  && \
+                  echo "$PRODUCT_ID" | grep -q "^GT-"
+               then _ChangeBuildType_ROG_
+               elif [ "$fwInstalledBaseVers" -ge 3006 ]  && [ "$isGNUtonFW" && \
```

Comment:
Syntax error on this line **6510**:

```
elif [ "$fwInstalledBaseVers" -ge 3006 ]  && [ "$isGNUtonFW" && \
```

Remove extraneous left square bracket `[` before `"$isGNUtonFW"`

Reply:
Great catch! Thank you :)

Reply:
Done!